gcp_variant_transforms/vcf_to_bq_common.py

```diff
@@ -83,19 +83,41 @@ def get_pipeline_mode(known_args):
     return PipelineModes.MEDIUM
   return PipelineModes.SMALL
 
+def _get_file_content(input_file):
```

Comment:
We should actually only read the lines from the header (it's possible for someone to just use a regular VCF file as 'representative'). This logic is actually in [vcf_header_parser](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/libs/vcf_header_parser.py), so we can reuse that code (with some refactoring) after #207 is merged?